src/client/components/Navigation/NavigationDesktop/NavigationDesktop.tsx

```diff
@@ -13,11 +15,17 @@ import NavAssessment from '../NavAssessment'
 
 const NavigationDesktop: React.FC = () => {
   const { i18n } = useTranslation()
+  const dispatch = useAppDispatch()
   const countryIso = useCountryIso()
   const assessment = useAssessment()
   const cycle = useCycle()
   const assessmentName = assessment.props.name
 
+  // Show navigation on first mount (ex. returing from Mobile view)
+  useEffect(() => {
+    dispatch(NavigationActions.updateNavigationVisible(true))
```

Comment:
#1305

Reply:
Is it visible only on desktop ?

Reply:
This configuration now means that, when you change the browser size to mobile, the navigation is hidden, when returning to desktop size it is shown again.
On both, you can still toggle the navigation